docs/feature-flagging/kill-switches.md

```diff
@@ -26,7 +26,7 @@ Here's how you would use the switch in your code:
 ```python
 # The argument user_id is required but doesn't affect assignment in this case.
 
-if (eppoClient.getStringAssignment(user_id, 'twilio-killswitch') == 'on'):
+if (eppoClient.getStringAssignment(user_id, 'twilio-killswitch', 'off') == 'on'):
```

Comment:
Could also turn this into a boolean flag

Reply:
Also the code suggests this should be Python code:

```
if eppo_client.get_boolean_assignment(user_id, 'twilio-killswitch', false):
```

Reply:
Good catch. I updated to Python syntax, but leaving it as a string flag (1) to match the screenshots, (2) True and False are somewhat ambiguous for a kill switch.